mopidy/backend.py

```diff
@@ -196,7 +245,7 @@ def prepare_change(self):
         """
         self.audio.prepare_change().get()
 
-    def translate_uri(self, uri):
+    def translate_uri(self, uri: str) -> Optional[str]:
```

Comment:
Why are these types `str` and not `Uri`?

Reply:
Just an oversight, I believe. I'll fix.